ship.config.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-commonjs */
+const fs = require('fs');
+const path = require('path');
+
+module.exports = {
+  mergeStrategy: { toSameBranch: ['master'] },
```

Comment:
This configures Ship.js to run only on `master` branch and merges the release PR back to `master`.

In other words, it won't run on any other branches, including this `chore/add-shipjs` branch.
So unfortunately, you cannot try this before merging this PR.

Reply:
> * write `.env` in your forked IS.js dir with the content `GITHUB_TOKEN=xxx`

Does Ship.js support `.env` files without locally installing `dotenv`? If not, it might be time to introduce `dotenv` to InstantSearch.js.

Reply:
(I tried in the experimental [Autocomplete.js](https://github.com/francoischalifour/autocomplete.js/blob/6885104d75be433fba27d709ff37bebbc7b20e3c/.env.sample) repo and it works without the `dotenv` dependency).

Reply:
@francoischalifour Ship.js comes with `dotenv` in it.
So, InstantSearch.js doesn't need it.